datasets/sorel-20m.yaml

```diff
@@ -20,7 +20,7 @@ Tags:
   - deep learning
   - labeled
   - machine learning
-License: See Terms of Use at "https://github.com/sophos-ai/SOREL-20M/blob/master/Terms%20and%20Conditions%20of%20Use.pdf"
+License: See Terms of Use at https://github.com/sophos-ai/SOREL-20M/blob/master/Terms%20and%20Conditions%20of%20Use.pdf
```

Comment:
Sorry, I should caught this earlier, didn't look at the formatting. You can use markdown to do the below.

See the [terms of use](https://github.com/sophos-ai/SOREL-20M/blob/master/Terms%20and%20Conditions%20of%20Use.pdf).

Reply:
Thanks; just tried updating it to markdown format!